src/ng-select/lib/ng-select.component.html

```diff
@@ -28,145 +26,140 @@
 			}
 		}
 
-        @if (multiple && multiLabelTemplate && selectedValues.length > 0) {
-            <ng-template [ngTemplateOutlet]="multiLabelTemplate"
-                         [ngTemplateOutletContext]="{ items: selectedValues, clear: clearItem }">
-            </ng-template>
-        }
+		@if (multiple && multiLabelTemplate && selectedValues.length > 0) {
+			<ng-template [ngTemplateOutlet]="multiLabelTemplate" [ngTemplateOutletContext]="{ items: selectedValues, clear: clearItem }">
+			</ng-template>
+		}
 
-        <div
-            [attr.aria-expanded]="isOpen"
-            [attr.aria-owns]="isOpen ? dropdownId : null"
-            aria-haspopup="listbox"
-            class="ng-input"
-            role="combobox">
-            <input
-                #searchInput
-                (blur)="onInputBlur($event)"
-                (change)="$event.stopPropagation()"
-                (compositionend)="onCompositionEnd(searchInput.value)"
-                (compositionstart)="onCompositionStart()"
-                (focus)="onInputFocus($event)"
-                (input)="filter(searchInput.value)"
-                [attr.aria-activedescendant]="isOpen ? itemsList?.markedItem?.htmlId : null"
-                [attr.aria-controls]="isOpen ? dropdownId : null"
-                [attr.id]="labelForId"
-                [attr.tabindex]="tabIndex"
-                [disabled]="disabled"
-                [readOnly]="!searchable || itemsList.maxItemsSelected"
-                [value]="searchTerm ? searchTerm : ''"
-                aria-autocomplete="list" />
-        </div>
-    </div>
+		<div [attr.aria-expanded]="isOpen" [attr.aria-owns]="isOpen ? dropdownId : null" aria-haspopup="listbox" class="ng-input">
```

Comment:
I think we do not need `[attr.aria-owns]="isOpen ? dropdownId : null"` at all as `[attr.aria-controls]` replaces it. But if it is needed due to some older version compatibility, it shoud be in the same element with `combobox`, hence an `input` element.
Based on this paragraph in [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role):
"When a combobox's popup is displayed, ensure the [aria-controls](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls) attribute on the `combobox` element is set to the [id](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#id) of the popup `listbox`, `tree`, `grid`, or `dialog` element. This is how the relationship between the element with the `combobox` role and the popup it controls is indicated. **(Note: In older ARIA specs, this was `aria-owns` rather than `aria-controls`, so you may see `aria-owns` in older `combobox` implementations. The `aria-owns` in the code should be updated to `aria-controls`!)"**